src/App.js

```diff
@@ -1,25 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import s from './App.module.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+import EnterName from "./components/EnterName";
+
+
+
+const App = (props) => {
+    return <div className={s.mainField}>
+        <EnterName name={props.state.state.name}/>
```

Comment:
`props.state.state.name` looks too nested. Try to:
1) use object destructuring
2) reduce nesting so you don't have to use `state.state`

Reply:
fixed